nerfstudio/data/pixel_samplers.py

```diff
@@ -422,9 +462,20 @@ def sample_method(
             sub_bs = batch_size // (self.config.patch_size**2)
             half_patch_size = int(self.config.patch_size / 2)
             m = erode_mask(mask.permute(0, 3, 1, 2).float(), pixel_radius=half_patch_size)
```

Comment:
This is making me realize, instead of eroding, it might be better to just rejection sample patches across the image, rejecting if the whole patch isn't within the mask. Could be faster if the masks are large and the patches are small.

Reply:
That is probably true. I didn't really look into the functionality of the `PatchPixelSampler` yet but can find some time to do so later this week.

Reply:
Gotcha. That could also go in a separate PR for sure.